### DISCUSSION TITLE
Source of statements

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#379] [General]
* Some notes have subfields indicating a source (other than $5). Commonly, these are agencies used for content rating or other third parties. When the object of a triple is a literal, is there a way for us to assign a source without reifying the triple as a metadata work? 
* Do these relationships need to be described as RDF?
* Crystal suggestion: Append source using boilerplate language at the end of the string. These are not identified using URI's or any controlled vocabulary, right? So it's just putting the two strings together?

**Comment by GordonDunsire** [comment]
Strictly speaking, it is not good practice to mix meta-categories in a single string value (i.e. a value and value about the value). But notes is notes, and what constitutes 'good' practice is a long-standing problem in cataloguing collectives ;-)
